experiment/coverage/README.md

```diff
@@ -7,8 +7,8 @@ by comparing the e2e test log and swagger spec of k8s.
 
 Run e2e tests with API operation detail(-v=8 option):
 ```
-$ go run hack/e2e.go -- -v --test \
-  --test_args="--ginkgo.focus=[Conformance] -v=8" > api.log
+$ ${GOPATH}/bin/kubetest --test \
```

Comment:
$GOPATH/bin is usually in $PATH already. you also need to have `go install k8s.io/test-infra/kubetest` prior to this, so i'm not sure how necessary this is. /shrug

Reply:
hehe, yeah I know that should be a common setting.
But my environment was not TBH, I just need to learn golang more.
will remove $GOPATH/bin :-)